frontend/src/components/Activity.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+function Activity(props) {
+    return (
+        <div className="column" key={props.id}>
```

Comment:
let's make this Component not have the `column` class. the idea is not to decide here where this component is going to be used. it might be used inside a column, or somewhere else.

then in Home.js, wrap the usage of this Component inside a div with the column class. that way we don't couple this implementation to the columns structure.